doc/source/cookbook.rst

```diff
@@ -1318,3 +1318,96 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Categorical Data
+----------------
+
+Using categorical data type to store data more efficiently and consistently in multiple DataFrames
+
+Create two DataFrames with one column each from a subset of the unique values with overlap. Append the dataframes and examine the memory usage.
+
+.. ipython:: python
+
+   from pandas.api.types import CategoricalDtype
+
+   domain = np.array(['A','B','C','D','E','F','G','H','I','J','a','b','c','d','e','f','g','h','i','j'])
+
+   df1 = pd.DataFrame({'A': domain.take(np.random.randint(0,12,size=100000))})
+   df2 = pd.DataFrame({'A': domain.take(np.random.randint(8,20,size=100000))})
+
+   df3 = df1.append(df2)
+   df3.memory_usage()
+
+.. ipython:: python
```

Comment:
if you are going to put text in between the diferent code samples it makes sense to have them in separate ipython blocks, otherwise it doesn't